### PR TITLE
Update metadata properties.file.uri on upload, var name correction

### PR DESCRIPTION
**Pull request by HZfive**

### CODE REVIEW

**Comment by stegaBOB** [comment]
Looks good! Can you turn this into a method to reduce the code duplication? Thanks!

**Comment by stegaBOB** [comment]
Looks good! Can you turn this into a method to reduce the code duplication? Thanks!

**Comment by HZfive** [comment]
> Looks good! Can you turn this into a method to reduce the code duplication? Thanks!

ready to go, upload a candy with 10 assets and everythings looks good

**Comment by stegaBOB** [comment]
Looks good to me! Thanks @HZfive

File: js/packages/cli/src/helpers/upload/pinata.ts [hunk 1]
```diff
@@ -33,7 +34,7 @@ export async function pinataUpload(
 ) {
   const gatewayUrl = gateway ? gateway : `https://ipfs.io`;
 
-  const manifestJson = JSON.parse(manifestBuffer.toString('utf8')); //JSON.parse(fs.readFileSync(manifestBuffer, 'utf-8'));
+  // const manifestJson = JSON.parse(manifestBuffer.toString('utf8')); //JSON.parse(fs.readFileSync(manifestBuffer, 'utf-8'));
```
Comment (stegaBOB) [inline js/packages/cli/src/helpers/upload/pinata.ts:37]:
```suggestion

```

**Review by stegaBOB — APPROVED** [review]

**Comment by HZfive** [comment]
add vercel.json for not deploy metaplex web

**Pull request merged**